CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v0.16.0
```

Comment:
This is a breaking change. Should this be v1? Or maybe we can at least mention it in the changelog that this is going to cause trouble for anyone who relies on `chrom_names` and `chrom_sizes`.

Reply:
I'll mention it in the changelog. There should be a v1 but I'm not sure that this is the moment to declare it.